src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentPlanCard/FreePlanCard/FreePlanCard.jsx

```diff
@@ -100,10 +112,21 @@ PlanUpgrade.propTypes = {
 }
 
 function FreePlanCard({ plan, scheduledPhase }) {
-  const { provider } = useParams()
-
+  const { provider, owner } = useParams()
+  const { codecovTrialMvp } = useFlags({
+    codecovTrialMvp: false,
+  })
   const { data: ownerData } = usePlanPageData()
+  const { data: planData } = usePlanData({
+    provider,
+    owner,
+    opts: { enabled: codecovTrialMvp },
+  })
+
   const uploadsNumber = ownerData?.numberOfUploads
+  const trialOngoing =
+    isTrialPlan(planData?.plan?.planName) &&
```

Comment:
Didn't we have a utility somewhere that did this function? I think in one of the 1st PRs you made for the usePlanData hook

Reply:
No that was removed